db/migrate/20170312171020_initialize_default_contexts.rb

```diff
@@ -0,0 +1,19 @@
+class InitializeDefaultContexts < ActiveRecord::Migration[5.0]
+  class PublishingContext < ActiveRecord::Base; end
+
+  def up
+    PublishingContext.create!(name: "milk_quantity")
+    PublishingContext.create!(name: "milk_quality")
+    PublishingContext.create!(name: "movement")
+    PublishingContext.create!(name: "temperature")
+    PublishingContext.create!(name: "intake")
+    PublishingContext.create!(name: "birth")
+    PublishingContext.create!(name: "calf")
+    PublishingContext.create!(name: "noise")
+    PublishingContext.create!(name: "health")
```

Comment:
I would not set up static data with migrations - we can use seeds for development and for production I would suggest to use just the normal web interface